src/current/_includes/releases/v23.2/v23.2.13.md

```diff
@@ -0,0 +1,88 @@
+      - PubSub
+      - SQL
+
+      [#130664][#130664]
+
+- The new [metric]({% link v23.2/metrics.md %}) `changefeed.total_ranges` allows observation of the number of changes that are watched by a changefeed aggregator. It uses the same polling interval as `changefeed.lagging_ranges`, which is controlled by the changefeed option `lagging_ranges_polling_interval`. [#130984][#130984]
+- The following groups of [metrics]({% link v23.2/metrics.md %}) and [logs]({% link v23.2/logging.md %}) have been renamed to include the buffer they are associated with. The previous metrics are still maintained for backward compatibility.
+      - `changefeed.buffer_entries_mem.*`
+      - `changefeed.buffer_pushback_nanos.*`
+
+      [#131417][#131417]
+- Timers have been added to key parts of the [changefeed]({% link v23.2/change-data-capture-overview.md %}) pipeline to help debug problems with feeds. [Metrics]({% link v23.2/metrics.md %}) `changefeed.stage.{stage}.latency` now emit latency histograms for each stage. The changefeed `scope` label allows debugging of a specific changefeed. [#131428][#131428]
+
+<h3 id="v23-2-13-operational-changes">Operational changes</h3>
+
+- The new [metric]({% link v23.2/metrics.md %}) `ranges.decommissioning` shows the number of ranges with a replica on a [decommissioning node]({% link v23.2/node-shutdown.md %}). [#130251][#130251]
+- The following new [metrics]({% link v23.2/metrics.md %}) show the number of RPC TCP connections established to remote nodes:
+      - `rpc.connection.connected`: the number of gRPC TCP level connections established to remote nodes.
+      - `rpc.client.bytes.egress`: the number of TCP bytes sent over gRPC on connections initiated by the cluster.
+      - `rpc.client.bytes.ingress`: the number of TCP bytes received over gRPC on connections initiated by the cluster.
+
+      [#130521][#130521]
+- Added a new configuration parameter, `server.cidr_mapping_url`, which maps IPv4 CIDR blocks to arbitrary tag names. [#130528][#130528]
+- The [metrics]({% link v23.2/metrics.md %}) `sql.bytesin` and `sql.bytesout` are now aggregate metrics if child metrics are enabled. [#130528][#130528]
+- The following new [metrics]({% link v23.2/metrics.md %}) track the number of bytes sent by an individual [changefeed]({% link v23.2/change-data-capture-overview.md %}) to each sink:
+    - `changefeed.network.bytes_in`
+    - `changefeed.network.bytes_out`
+
+    [#130664][#130664]
+- You can now set the log format for the `STDERR` changefeed sink using the `format` field in the `stderr` sink section of the [logging]({% link v23.2/logging.md %}) configuration. [#131533][#131533]
+
+<h3 id="v23-2-13-db-console-changes">DB Console changes</h3>
+
+- The [DB Console]({% link v23.2/ui-overview.md %}) now shows an alert message if the cluster's [enterprise license]({% link v23.2/licensing-faqs.md %}) is expired or will expire in fewer than 15 days, or if the cluster has no enterprise license set. [#130425][#130425]
```

Comment:
Our edit from my PR for consideration https://github.com/cockroachdb/docs/blob/1e52565e2fb84766836bc1824435d7351d4ad6e5/src/current/_includes/releases/v24.2/v24.2.4.md?plain=1#L33